### DISCUSSION TITLE
3.1.3

### DISCUSSION THREAD

**Discussion by sfmskywalker** [elsa-workflows/elsa-core#5284] [Design & tech]
## What's Changed
* bumped versions to fix dependency vulnerabilities by @stevetayloruk in https://github.com/elsa-workflows/elsa-core/pull/5256

## New Contributors
* @stevetayloruk made their first contribution in https://github.com/elsa-workflows/elsa-core/pull/5256

**Full Changelog**: https://github.com/elsa-workflows/elsa-core/compare/3.1.2...3.1.3

<hr /><em>This discussion was created from the release <a href='https://github.com/elsa-workflows/elsa-core/releases/tag/3.1.3'>3.1.3</a>.</em>

**Comment by glime-ai** [comment]
The changes in the 3.1.3 release of Elsa Workflows include bumping up versions to fix dependency vulnerabilities. This update was contributed by @stevetayloruk, and it was their first contribution to the project. You can see the full changelog by comparing the differences between the 3.1.2 and 3.1.3 tags in the repository.

For more details, you can visit the release page directly at: https://github.com/elsa-workflows/elsa-core/releases/tag/3.1.3

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
